src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -111,6 +111,9 @@
   <Node Name="BoundDisposableValuePlaceholder" Base="BoundValuePlaceholderBase">
   </Node>
 
+  <Node Name="BoundLValuePlaceholder" Base="BoundValuePlaceholderBase">
```

Comment:
>BoundLValuePlaceholder [](start = 14, length = 22)

It looks like there is a special handling for BoundImplicitReceiver in CSharpOperationFactory. We are using this new node in place where BoundImplicitReceiver was used. It feels like adjustments should be made to CSharpOperationFactory.